controllers/ingressController.go

```diff
@@ -377,33 +344,26 @@ func GetActiveLeases(ns string, name string) []models.Leases {
 }
 
 //DeleteLeases ...
-func DeleteLeases(ns string, name string, ip string, ID uint) (bool, int, error) {
-	updateStatusflag, dbflag := true, false
-	var err error
-	errs := 0
+func DeleteLeases(ns string, name string, ip string, ID uint) (ingress_driver.DisableUserResponse, error) {
 	if database.DB == nil {
 		database.Conn()
 	}
 
-	for _, kubeClient := range config.KubeClients {
-		clientset := kubeClient.ClientSet
-		myclientset := pkg.MyClientSet{Clientset: clientset}
-		_, dbflag, err = myclientset.RemoveIngressIP(ns, name, ip)
-		if err != nil {
-			errs = errs + 1
-		}
-		if dbflag {
-			updateStatusflag = false
-		}
+	req := ingress_driver.DisableUserRequest{
```

Comment:
IMO this should be `RevokeLeaseRequest`. Since disableuser is very specific to looker

Reply:
done.